chef/cookbooks/crowbar/templates/default/crowbar-upgrade-os.sh.erb

```diff
@@ -145,7 +146,7 @@ initiate_node_upgrade()
     rm -f /etc/nova/nova.conf.d/200-crowbar-upgrade.conf
     rm -f /etc/neutron/neutron-openvswitch-agent.conf.d/200-crowbar-upgrade.conf
 
-    if [ $is_openvswitch_updated == 0 ] && [ $was_openvswitch_enabled == 0 ]; then
+    if [ $is_openvswitch_installed == 0 ] && [ $was_openvswitch_enabled == 0 ]; then
```

Comment:
I think it would work well enough to have `if rpm --quiet -q openvswitch` here ... well but anyway it's temporary solution

Reply:
Indeed, because if the package is upgraded in iny other time, that check for `$was_openvswitch_enabled` will also fail

So it may be just easier to remove that part and just enable it if the package is installed